public/app/features/explore/LiveLogs.tsx

```diff
@@ -147,6 +159,10 @@ class LiveLogs extends PureComponent<Props, State> {
           </tbody>
         </table>
         <div className={styles.logsRowsIndicator}>
+          <Button variant="secondary" onClick={onClear} className={styles.button}>
```

Comment:
also  you can use the `icon` prop on the button to have the icon properly rendered as a prefix (and without the need to add the "margin" with `&nbsp;`)

Reply:
I wouldn't mind if we used `square-shape` icon without the type `mono` (that makes square filled) , as we already use `square-shape` without `mono` in live tailing button on top. So it would actually make it more consistent. 
<img width="136" alt="image" src="https://user-images.githubusercontent.com/30407135/228477598-467610d8-18be-4c33-932f-ba14180a9f97.png">